rbd.c

```diff
@@ -943,12 +945,14 @@ static void rbd_finish_aio_generic(rbd_completion_t completion,
 {
 	struct tcmu_device *dev = aio_cb->dev;
 	struct tcmulib_cmd *tcmulib_cmd = aio_cb->tcmulib_cmd;
-	struct iovec *iovec = tcmulib_cmd->iovec;
-	size_t iov_cnt = tcmulib_cmd->iov_cnt;
+	struct iovec *iov = aio_cb->iov;
+	size_t iov_cnt = aio_cb->iov_cnt;
 	uint32_t cmp_offset;
 	uint16_t asc_ascq;
 	int64_t ret;
 	int tcmu_r;
+	struct iovec *iovec;
+	size_t iov_cnt;
 
```

Comment:
Note: I fixed this in commit:

https://github.com/open-iscsi/tcmu-runner/commit/d1422948d0285c3453fff5f43abf4ebe6459ff82